beacon-chain/db/filesystem/blob.go

```diff
@@ -261,6 +256,9 @@ func (bs *BlobStorage) Prune(currentSlot primitives.Slot) error {
 			if err != nil {
 				return err
 			}
+			if err := bs.processFolder(folder, currentSlot, bs.retentionSlots); err != nil {
```

Comment:
Seems like this edit got scrambled. Delete 259-261 and change `num, err := bs.processFolder(folder, currentSlot, bs.retentionSlots)` above.